firebase.json

```diff
@@ -24,1610 +24,648 @@
+        "source": "/:lang/docs/design/responsive/art_direction",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/develop/style_and_layout/art_direction?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/docs/design/visual_story?(.html)",
-        "destination": "/docs/getting_started/visual_story.html",
+        "source": "/:lang/docs/design/responsive/control_layout",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/develop/style_and_layout/control_layout?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/docs/tutorials/visual_story/add_more_pages?(.html)",
-        "destination": "/docs/getting_started/visual_story/add_more_pages.html",
+        "source": "/:lang/docs/design/responsive/custom_fonts",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/develop/style_and_layout/custom_fonts?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/docs/design/visual_story/add_more_pages?(.html)",
-        "destination": "/docs/getting_started/visual_story/add_more_pages.html",
+        "source": "/:lang/docs/design/responsive/placeholders",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/develop/style_and_layout/placeholders?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/docs/tutorials/visual_story/animating_elements?(.html)",
-        "destination": "/docs/getting_started/visual_story/animating_elements.html",
+        "source": "/:lang/docs/design/responsive/responsive_design",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/develop/style_and_layout/responsive_design?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/docs/design/visual_story/animating_elements?(.html)",
-        "destination": "/docs/getting_started/visual_story/animating_elements.html",
+        "source": "/:lang/docs/design/responsive/style_pages",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/develop/style_and_layout/style_pages?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/docs/tutorials/visual_story/congratulations?(.html)",
-        "destination": "/docs/getting_started/visual_story/congratulations.html",
+        "source": "/:lang/docs/design/responsive_amp",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/develop/style_and_layout/?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/docs/design/visual_story/congratulations?(.html)",
-        "destination": "/docs/getting_started/visual_story/congratulations.html",
+        "source": "/:lang/docs/fundamentals/",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/?referrer=ampproject.org",
```

Comment:
should this go to `start`?

Reply:
We don't have bucket landing pages yet - I think Paul wasn't a fan of them: https://amp.dev/documentation/guides-and-tutorials/start/create.html also maybe?

Reply:
Hm, let's leave as it is then